wp-admin/js/widgets/media-image-widget.js

```diff
@@ -141,11 +165,6 @@
 				url: control.model.get( 'url' )
 			};
 
-			wp.media.events.trigger( 'editor:image-edit', {
-				metadata: metadata,
-				image: control.$el.find( 'img:first' )
-			} );
-
```

Comment:
I could find no purpose for this, other than being copy/pasted from the another place in core.